pymeasure/instruments/rohdeschwarz/fsseries.py

```diff
@@ -133,7 +137,7 @@ def continue_single_sweep(self):
     trace_mode = Instrument.control(
         "DISP:TRAC:MODE?",
         "DISP:TRAC:MODE %s",
-        "Control trace mode ('WRIT', 'MAXH', 'MINH', 'AVER' or 'VIEW')",
+        "Control trace mode ('WRITe', 'MAXHold', 'MINHold', 'AVERage' or 'VIEW')",
```

Comment:
Attention: Below, you only allow "WRIT", but here you suggest, that the correct value would be "WRITe", such that a user will always enter invalid values (except for "VIEW").
Either make the long version allowed in the values entry, or state, that you have to enter the upper case letters for a mode.